src/components/search/search.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SearchView } from './search';
+
+const noop = jest.fn();
+
+describe('Search', () => {
```

Comment:
For components tests it is better to use the following template for tests declaration: `{Component_name} component tests`. In your case it would look like: `Search component tests`